src/main/java/com/github/dockerjava/api/model/HostConfig.java

```diff
@@ -172,7 +172,7 @@
      * @since {@link RemoteApiVersion#VERSION_1_20}
      */
     @JsonProperty("SecurityOpt")
-    private List<String> securityOpts;
+    private String[] securityOpts;
```

Comment:
It's been a while, but I think I did that because of some json serialization glitch.  So I made CreateContainerCmd.getSecurityOpts() match the other getters (e.g. CreateContainerCmd.getEnv()).  I guess HostConfig doesn't need to match, strictly speaking.  I can make HostConfig use List<String> and CreateContainerCmd use String[] if that's better in some way.